pkg/ovsdb/transact.go

```diff
@@ -1114,7 +1117,7 @@ func doDelete(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 func preWait(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.OperationResult) error {
 	if ovsOp.Timeout != 0 {
 		err := fmt.Errorf("only support timeout 0")
-		ovsResult.Error = err.Error()
+		*ovsResult.Error = err.Error()
```

Comment:
this error set should be done for all operations

Reply:
ok